image.yaml

```diff
@@ -73,17 +73,17 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_737_CR1
+                  ref: EAP_750_CR1
```

Comment:
This  should be `EAP_740_CR1` probably (there isn't any tag like `EAP_750_CR1`).